node_modules/col-canvas/lib/poller.js

```diff
@@ -464,9 +469,11 @@ var handleAssignmentCategory = function(ctx, users, categories, assignment, call
       return callback();
     });
 
-  // Create a category for an assignment if it doesn't exist yet
+  // Create a category for an assignment if it doesn't exist yet.
+  // By default, the category will be marked as not visible
   } else {
-    CategoriesAPI.createCategory(ctx, assignment.name, assignment.id, assignment.name, function(err, newCategory) {
+    var enableAssignmentCategories = opts.enableAssignmentCategories  || false;
```

Comment:
By default, a category will be created with a visibility of `true`. However, when a category is created from a Canvas assignment by the poller, its visibility is set to `false` by default. Therefore, the line above is necessary to ensure that `enableAssignmentCategories` isn't set to `null`, as the category would then be created as `visible`